src/styles/blocks/container.scss

```diff
@@ -0,0 +1,25 @@
+.container {
+  display: flex;
+  justify-content: center;
+  margin-top: 50px;
+
+  &__cards {
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 4 * $cardWidth + 3 * $cardMargin;
+    justify-content: center;
+  }
+
+  &__card {
+    margin-right: $cardMargin;
+    margin-bottom: 46px;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-child(-n + 4) {
+      margin-bottom: 0;
+    }
+  }
+}
```

Comment:
Move this code to the separate `cards.scss` file and nest it to cards block there.

Now you  increase selectors specificity, but it isn't necessary there.

In css your 13 line would looks like:
`.container__cards .cards__card` in new file you would have only `.cards__card`.